Foundation/ExtraStringAPIs.swift

```diff
@@ -13,19 +13,9 @@
 // Random access for String.UTF16View, only when Foundation is
 // imported.  Making this API dependent on Foundation decouples the
 // Swift core from a UTF16 representation.
-extension String.UTF16View.Index : Strideable {
```

Comment:
In corelibs-foundation... I can't even estimate, but the guess is that it's pretty minor. There is waaaay less Swift code that uses corelibs-foundation than there is code that uses Foundation.